examples/sample-with-dns-zone.tfvars

```diff
@@ -0,0 +1,237 @@
+#-----------------------------------------------------------------------------
+
+# Name of the A record, of following dns zone, where the ecl watch ip is placed
+# This A record will be created and therefore should not exist in the following
+# dns zone.
+
+a_record_name="hpcc"
+
+#-----------------------------------------------------------------------------
+
+# Name of an existing dns zone.
+
+dns_zone_name="<name of existing dns zone>"
```

Comment:
Same as above:  Empty string for the actual value, but put the example into the description.